### DISCUSSION TITLE
Agent disappeared from agents list after submitting the report

### DISCUSSION THREAD

**Discussion by shiraznave** [yasserfarouk/scml#105] [one-shot-qa] [answered]
Hello @yasserfarouk,
Today me and my team submitted our report (after submitting our code last week).
Before we submitted our report, out agent's name appeared on the list of the agents (inside the leader board). After submitting it disappeared and we're concerned there might a problem.

Our agent name: AgentSAS. 

Thank you,
Shiraz

**Comment by a4rcvv** [comment]
Hello @shirazbar . Sorry that this is not an answer, but I want to report about this problem.

I'm facing the same problem. Also, I found that both my agent (AgentNeko) and AgentSAS are appeared in "Running in the official competition" field in the **Standard** and **Collusion** track, not in the OneShot track.

I opened a new discussion (#106) to report on broader issues.

**Comment by yasserfarouk** [comment]
This problem should be resolved now. Please reply if you are still having an issue.

**Comment by yasserfarouk** [comment, marked as answer]
Sorry for this issue. We will fix it soon. Please note that a confirmation
email will be sent to every participant after the report deadline with the
official track in which they are participating. Even if a mistake happens
at this stage, you can simply reply with the correction.

The display issue on the website will be fixed asap.

Finally, please confirm that your agent has the checkbox for oneshot
checked and for standard and collusion unchecked in the Active Agent tab.

Thanks
yasser
